components/speedreader/resources/panel/theme/dark.ts

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import ITheme from 'brave-ui/theme/theme-interface'
+ import defaultDarkTheme from 'brave-ui/theme/brave-dark'
+ import IThemeShields from './theme'
+
+ const darkTheme: ITheme & IThemeShields = {
+   ...defaultDarkTheme,
+   name: 'Speedreader Dark',
+   color: {
+     ...defaultDarkTheme.color,
+     background01: '#1E2029'
```

Comment:
perhaps don't override default Theme colors. This would be if you wanted a child control to take on different colors. Instead, you can just add a new variable to your local theme interface, which is called here `IThemeShields` and should ideally be renamed to something like `IThemeSpeedreader`

Reply:
moving to css variable